dandi/validate_types.py

```diff
@@ -1,35 +1,53 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
-from enum import Enum
+from enum import Enum, IntEnum
 from pathlib import Path
+from typing import Any
 
 
 @dataclass
 class ValidationOrigin:
-    name: str
-    version: str
-    bids_version: str | None = None
+    name: str  # Validator name
+    version: str  # Validator version
+    standard: str | None = (
+        None  # Standard being validated against # TODO: Enum for the standards??
```

Comment:
@yarikoptic We can create an enum class to define the supported standards. Do you know what those those standards are ATM?

Reply:
Sure. For starters:

- BIDS
- NWB
- HED
- OME-ZARR

Could be also at the level of the "file format" standard:

- JSON
- YAML
- TSV

Reply:
might want also to create Enum for validators since we also have a set of them.

Reply:
I added the standard `ZARR` as well since you have already started using it.

Reply:
Marking this as resolved since the new `Enum` subclasses have been reviewed in a meeting.